conda-forge.yml

```diff
@@ -1,5 +1,5 @@
 appveyor:
   secure: {BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1}
-provider: {linux_aarch64: default, linux_ppc64le: default, win: azure}
```

Comment:
did it not finish on drone?

Reply:
there was an error on drone: https://cloud.drone.io/conda-forge/conda-package-handling-feedstock/2/1/2

Reply:
Can you revert this now?